doc/sources/.static/fresh.css

```diff
@@ -1,641 +1,938 @@
-		padding-right: 1em;
-	}
+.highlight .nc {
+    color: var(--highlight-nc)
+}
+
+.highlight .no {
+    color: var(--neutral-secondary-text)
+}
+
+.highlight .nd {
+    color: var(--highlight-go)
+}
+
+.highlight .ni {
+    color: var(--highlight-nc)
+}
+
+.highlight .ne {
+    color: var(--highlight-gr)
+}
+
+.highlight .nf,.highlight .nl {
+    color: var(--highlight-nc)
+}
+
+.highlight .nn,.highlight .nx,.highlight .py {
+    color: var(--neutral-secondary-text)
+}
+
+.highlight .nt {
+    color: var(--versionchanged-color)
+}
+
+.highlight .nv,.highlight .ow,.highlight .w {
+    color: var(--neutral-secondary-text)
+}
 
-	.toc {
-		width: auto;
-		background-color: #f5f5f5;
-		border: 1px solid #e0e0e0;
-		padding: 10px;
-		float: right;
-		margin: 0 0 10px 10px;
-		font-size: 0.9em;
-	}
+.highlight .w {
+    text-decoration-color: currentcolor
+}
+
+.highlight .mb,.highlight .mf,.highlight .mh,.highlight .mi,.highlight .mo {
+    color: var(--highlight-m)
+}
+
+.highlight .sa {
+    color: var(--highlight-s)
+}
+
+.highlight .sb,.highlight .sc {
+    color: var(--highlight-sb)
+}
+
+.highlight .dl {
+    color: var(--highlight-s)
+}
+
+.highlight .sd {
+    color: var(--secondary-text-color)
+}
+
+.highlight .s1,.highlight .s2,.highlight .se,.highlight .sh,.highlight .si,.highlight .sr,.highlight .ss,.highlight .sx {
+    color: var(--highlight-s)
+}
+
+.highlight .bp {
+    color: var(--highlight-bp)
+}
+
+.highlight .fm {
+    color: var(--highlight-nc)
+}
+
+.highlight .vc,.highlight .vg,.highlight .vi,.highlight .vm {
+    color: var(--neutral-secondary-text)
+}
+
+.highlight .il {
+    color: var(--highlight-m)
+}
+
+@media (min-width:1024px) {
+    #toplogo {
+        width: 20%;
+        height: 80px;
+        display: flex;
+        justify-content: flex-start
+    }
+
+    #topmenu {
+        width: 80%;
+        height: 80px
+    }
+
+    #topbar ul {
+        justify-content: flex-end
+    }
+
+    #topmenu ul.navigation li {
+        margin-right: 3em
+    }
+
+    div.sphinxsidebar {
+        width: 260px;
+        position: sticky;
+        top: 0;
+        height: 80vh;
+        padding-top: 1em
+    }
+
+    #content {
+        width: calc(100% - 260px)
+    }
+
+    .sphinxsidebarwrapper {
+        display: flex;
+        flex-direction: column;
+        height: 100%
+    }
+
+    .api-index {
+        height: 100px;
+        display: flex;
+        flex-grow: 1
+    }
+
+    .is-api .sphinxsidebarwrapper .current {
+        display: flex;
+        flex-grow: 1;
+        flex-direction: column
+    }
+
+    #content>.wrapper {
+        padding-left: 1em;
+        padding-right: 1em
+    }
+
+    .toc {
+        width: auto;
+        background-color: var(--bg-accent-color);
+        border: 1px solid var(--dark-border-color);
+        padding: 10px;
+        float: right;
+        margin: 0 0 10px 10px;
+        font-size: .9em
+    }
+
+    .toc ul li li li {
+    border-left-color: var(--dark-border-color);
+}
```

Comment:
Why can't be placed outside of the media query?

Reply:
its how it is currently 
https://github.com/kivy/kivy/blob/master/doc/sources/.static/fresh.css#L630
and i forgot to add 
https://github.com/kivy/kivy/blob/master/doc/sources/.static/fresh.css#L630

Reply:
nvm its there